homework/src/nyc/c4q/ahhhlvin/Domestic.java

```diff
@@ -0,0 +1,16 @@
+package nyc.c4q.ahhhlvin;
+/**
+ * Created by alvin2 on 4/29/15.
+ * Alvin Kuang
+ * C4Q Access Code 2.1
+ */
+public interface Domestic
+{
+    public void setName(String name);
```

Comment:
I just looked this up because I have been wondering about it and this is what I found: http://stackoverflow.com/questions/161633/should-methods-in-a-java-interface-be-declared-with-or-without-a-public-access-m

tl;dr You don't have to specify that these methods are public because a class method that implements an interface is always public.